### ISSUE TITLE
tabindex

### ISSUE THREAD

**Issue by jeanpan** [jeanpan/notes#10]
The `tabindex` global attribute indicates if its element can be focused, and if it participates in sequential keyboard navigation. 

|                       | Keyboard navigation | `focus()` |                                                                                                                                                                                                       |
| --------------------- | ------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `tabindex="-1"` | No                  | Yes       | A `negative` value means that element should be **focusable**, but should not be reachable via sequential keyboard navigation. <br>It’s mostly useful to create accessible widgets with JS `focus()`. |
| `tabindex="0"`  | Yes                 | Yes       | The `0` means that the element should be **focusable** in sequential keyboard navigation and it’s order is defined by document’s source order.                                                        |
| `tabindex="1"`  | Yes                 | Yes       | A `positive` value means the element should be focusable in sequential keyboard navigation with its order defined by the value of number.                                                             |
